dapp/src/sections/AuctionCarousel.tsx

```diff
@@ -0,0 +1,178 @@
+                const newItem = auctions[i % auctions.length];
+                const key = `${newItem.name}__${i}`;
+                return { ...newItem, key };
+            });
+        }
+
+        return auctions.map((auction) => ({ ...auction, key: auction.name }));
+    })();
+
+    useEffect(() => {
+        const el = containerRef.current;
+        if (!el) return;
+
+        const ro = new ResizeObserver((entries) => {
+            const w = entries[0]?.contentRect.width ?? el.offsetWidth;
+            setContainerWidth(w);
+            if (!hasMeasured) setHasMeasured(true);
+        });
+        ro.observe(el);
+        return () => ro.disconnect();
+    }, [hasMeasured]);
+
+    return (
+        <section className="container py-14 md:py-20" ref={containerRef}>
+            <div className="w-full">
+                <AuctionCarouselHeader />
+
+                {isPreparing ? (
+                    <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-4">
+                        {Array.from({ length: 4 }).map((_, index) => (
+                            <div key={index} className="animate-pulse">
+                                <div className="bg-names-neutral-12 rounded-lg h-64"></div>
+                            </div>
+                        ))}
+                    </div>
+                ) : !hasAuctions ? (
+                    <div className="text-center py-12">
+                        <p className="text-body-lg text-names-neutral-70">
+                            No active auctions at the moment
+                        </p>
+                    </div>
+                ) : (
+                    <div className="relative">
+                        <div className="absolute top-0 left-0 w-[60px] h-full bg-gradient-to-r from-[#0b0c23] via-[#0b0c23cc] to-transparent pointer-events-none z-10"></div>
+                        <div className="absolute top-0 right-0 w-[60px] h-full bg-gradient-to-l from-[#0b0c23] via-[#0b0c23cc] to-transparent pointer-events-none z-10"></div>
+
+                        {/* Carousel with autoplay that pauses on hover and when tab is not visible */}
+                        {/* <Carousel autoPlay pauseOnHover items={auctions} renderItem={renderItem} /> */}
+
+                        <Swiper
+                            slidesPerView={slidesPerView}
+                            autoplay={{
+                                // disableOnInteraction: true,
+                                delay: 2500,
+                                waitForTransition: true,
+                                pauseOnMouseEnter: true,
+                            }}
+                            centeredSlides
+                            loop
+                            className="mySwiper"
+                            modules={[Autoplay]}
+                        >
+                            {auctionsToRender.map((auction) => (
+                                <SwiperSlide key={auction.key}>
```

Comment:
```suggestion
                                <SwiperSlide key={auction.key} className="pb-xs">
```
Could we add some bottom padding so the cards’ shadows are visible?

Reply:
added. Nice, really didn't notice that 👀